spec/requests/api/v1/third_spaces/third_spaces_request_spec.rb

```diff
@@ -118,20 +118,24 @@
   end
 
   it "can create a new third space" do
+    json_tags = JSON.generate(['happy', 'studious'])
+    json_photos = JSON.generate([])
+    json_hours = JSON.generate("Olive Hoyl")
+
     space_params = ({ id: 126,
                     yelp_id: "Juana Bea",
                     name: "Barton Inc",
                     address: "8761 DuBuque Lights",
                     rating: 37.47,
                     phone: "7842541779",
-                    photos: [],
+                    photos: json_photos,
```

Comment:
Thank you for fixing this, I wasn't sure what was the best way to do it. We can fix it tomorrow once all functionality is good to go.